ehitchhiking-ui/src/app/shared/components/notification/notification.component.sass

```diff
@@ -1,21 +1,22 @@
 @import '../../../../styles/colors'
 @import '../../../../styles/fonts-constants'
+@import '../../../../styles/constants'
 
 .notification
   &-container
     color: $white
-
+    margin-top: $default-offset * 3
```

Comment:
I don't think that there is a good idea to use default-offset here. Pls set 3rem